decidim-core/lib/decidim/content_parsers/user_parser.rb

```diff
@@ -35,19 +35,17 @@ def metadata
         Metadata.new(existing_users)
       end
 
-      private
-
       def users
         @users ||=
           existing_users.index_by(&:nickname)
       end
 
       def existing_users
-        @existing_users ||= Decidim::User.where(organization: current_organization, nickname: content_nicknames)
+        @existing_users ||= Decidim::User.where("decidim_organization_id = ? AND LOWER(nickname) IN (?)", current_organization.id, content_nicknames)
```

Comment:
Isn't it redundant since the `content_nicknames` only contains downcased nicknames ?